### ISSUE TITLE
new term request: hemiplegic migraine

### ISSUE THREAD

**Issue by hannahblau** [obophenotype/human-phenotype-ontology#11759]
**Preferred term label:**
Hemiplegic migraine

**Synonyms**

**Definition (free text, please give PubMed ID)**
Migraine with aura consisting of fully reversible motor weakness accompanied by fully reversible visual, sensory and/or speech/language symptoms.

[PMID 29368949](https://pubmed.ncbi.nlm.nih.gov/29368949/)
_The International Classification of Headache Disorders, 3rd edition_ (2018)
.pdf available from
[Cephalalgia 38 (1) : 1-211](https://journals.sagepub.com/toc/cepa/38/1 )
see page 22 of the article for definition of hemiplegic migraine
The classification is also available in html form; definition of hemiplegic migraine is [here](https://ichd-3.org/1-migraine/1-2-migraine-with-aura/1-2-3-hemiplegic-migraine/)

[PMID 42633474](https://pubmed.ncbi.nlm.nih.gov/42633474/) says the motor weakness is unilateral, although this is not a requirement in the ICHD definition.
"It is characterised by transient unilateral motor weakness accompanied by headaches and other neurological symptoms such as paraesthesia, visual disturbances, or aphasia."

[PMID 39174245](https://pubmed.ncbi.nlm.nih.gov/39174245/) Chapter 9 of _Handbook of Clinical Neurology_ says "Hemiplegic migraine consists of attacks of migraine with aura that includes reversible motor weakness."

**Parent term (use hpo.jax.org/app)**
HP:0002077 Migraine with aura
Nota bene: to make Hemiplegic migraine a subclass of HP:0002077, you would have to modify the definition of HP:0002077 to admit the possibility that an aura can take the form of motor weakness, as this is exactly what distinguishes hemiplegic migraine. Currently, HP:0002077 definition says "The symptoms of an aura may include fully reversible visual, sensory, and speech symptoms but not motor weakness." Nonetheless, every reference I could locate considers hemiplegic migraine to be a subtype of migraine with aura. (I can open a separate issue if you like for this revision.)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
MIGRAINE, FAMILIAL HEMIPLEGIC, 1; FHM1 [OMIM #141500](https://omim.org/entry/141500)
MIGRAINE, FAMILIAL HEMIPLEGIC, 2; FHM2 [OMIM #602481](https://omim.org/entry/602481)
MIGRAINE, FAMILIAL HEMIPLEGIC, 3; FHM3 [OMIM #609634](https://omim.org/entry/609634)

**Your nano-attribution (ORCID)**
0000-0003-4557-5492
